packages/orbit-components/src/Itinerary/ItinerarySegment/ItineraryIcon.tsx

```diff
@@ -100,17 +38,35 @@ const ItinerarySegmentStopIcon = ({ type, children }: Props) => {
   const { index, last, isPrevHidden, isHidden, count } = usePart();
 
   return (
-    <IconStyled
-      index={index}
-      last={last}
-      isPrevHidden={isPrevHidden}
-      isHidden={isHidden}
-      count={count}
+    <div
```

Comment:
I believe there's a problem with the z-index here, as some of the vertical lines are not showing:

![image](https://github.com/kiwicom/orbit/assets/8863238/9ca204da-ee9b-4a6b-aed4-b2391d09c7b1)


https://kiwicom-orbit-tw-itinerary.surge.sh/?path=/story/itinerary--inside-modal&globals=backgrounds.grid:!false&knob-city=Prague&knob-date=Fr,%2019.10&knob-station=V%C3%A1clav%20Havel%20Airport%20Prague%20(PRG)&knob-time=14:05&knob-type=warning

Reply:
Well spotted, it was missing a `z-default` class on the main div of this component. Added.

Reply:
I guess surge didn't deploy the latest version? I don't see it fixed.

Reply:
Now I can see it 💯